README.md

```diff
@@ -30,9 +30,16 @@ Here's how to use it in a playbook:
     - xfce
 ```
 
+## New Repositories from a Skeleton ##
+
+Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+for step-by-step instructions on how to start a new repository from
+a skeleton. This will save you time and effort when configuring a
+new repository!
+
```

Comment:
We're removing this on non-skeletons, right?

Reply:
See commit 66b35ee.